build_test/lib/src/test_bootstrap_builder.dart

```diff
@@ -32,10 +33,41 @@ class TestBootstrapBuilder extends Builder {
     var assetPath = id.pathSegments.first == 'lib'
         ? p.url.join('packages', id.package, id.path)
         : id.path;
+
+    var vmRuntimes = [Runtime.vm];
+    var browserRuntimes =
+        Runtime.builtIn.where((r) => r.isBrowser == true).toList();
+    var nodeRuntimes = [Runtime.nodeJS];
+    if (await buildStep.canRead(AssetId(id.package, 'dart_test.yaml'))) {
+      var config = load('dart_test.yaml');
```

Comment:
Can we find a way to do this only once, instead of once per test file? Maybe it can be a `Resource`?

Reply:
Sure, a Resource would make sense.

Reply:
Ended up not needing a resource as there is nothing worth bothering cleaning up explicitly like extra processes, temp dirs, etc.